packages/docs/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "composite": true,
```

Comment:
Is composite correct here? I don’t think we have a top level tsconfig at the moment

Reply:
Well, it's used for when you have a primarily TypeScript monorepo structure. I think it's safe to stay.

Reply:
https://www.typescriptlang.org/tsconfig#composite

Not sure if it's actually doing anything here though

Reply:
In this scenario we're enforcing that:
- declarations must be generated for each build (so ts packages including any given package will receive the types)
- all files must be in the `include` array
- and that the rootDir defaults to the child package (top-level tsconfig)

This is all done by `composite`

Reply:
And... further research shows that declarations aren't generated despite it being on the handbook..

Reply:
Composite was not required